### PR TITLE
Create HANDBOOK.md

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
### Context
Me earlier today:
```
[1:10 PM]Ethan (he/Webmaster): TODO: LC dailies handbook. Prior art: https://acmcsuf.com/shorter-handbook
```
### References
- <https://acmcsuf.com/shorter-handbook>

### CODE REVIEW

File: docs/HANDBOOK.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### Operate with confidence
+
+The software is a
+[Discord slash command](https://discord.com/developers/docs/interactions/application-commands).
+All that is required to use the tool is a Discord account with the `Verified`
+role on the [ACM CSUF Discord server](https://acmcsuf.com/discord).
+
+- This slash command is only available to members with the `Verified` role.
+- This slash command is available in all text channels of the
```
Comment (karnikaavelumani) [inline docs/HANDBOOK.md:45]:
```suggestion
- This slash command is available in the `📚algo-chat` text channel of the
```

File: docs/HANDBOOK.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Wait for the response from the slash command to confirm the registration. This
+  may take a second.
+
+### Submit your Leetcode solution
+
+Once you have registered your Leetcode username, you are ready to participate in
+our weekly competitions.
+
+- Type `/lc submit` in the `📚algo-chat` text channel to use the slash command.
+- Populate the required field `lc_submission_url` with the URL to your accepted
```
Comment (karnikaavelumani) [inline docs/HANDBOOK.md:66]:
```suggestion
- Populate the required field `submission_url` with the URL to your accepted
```

File: docs/HANDBOOK.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  Leetcode solution of a daily challenge from the current season e.g. all
+  Leetcode dailies announced since the previous Sunday at midnight UTC.
+- Press <kbd>Enter</kbd> to submit the solution.
+- Wait for the response from the slash command to confirm the submission. This
+  may take a second.
+- If the submission was successful, your submission will be added to
+  [the leaderboard](https://lc-dailies.deno.dev/seasons/latest).
+
+|                           Valid `lc_submission_url` values                           |
```
Comment (karnikaavelumani) [inline docs/HANDBOOK.md:75]:
```suggestion
|                           Valid `submission_url` values                           |
```

**Review by karnikaavelumani — CHANGES_REQUESTED** [review]
Please review the changes!

File: docs/HANDBOOK.md (hunk 1 above)
Comment (EthanThatOneKid) [reply docs/HANDBOOK.md]: Thank you. Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/13/commits/7b28dae97f65bd76d98ff174ecf8c70252d99ec3>.

File: docs/HANDBOOK.md (hunk 2 above)
Comment (EthanThatOneKid) [reply docs/HANDBOOK.md]: Thank you. Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/13/commits/7b28dae97f65bd76d98ff174ecf8c70252d99ec3>.

File: docs/HANDBOOK.md (hunk 3 above)
Comment (EthanThatOneKid) [reply docs/HANDBOOK.md]: Thank you. Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/13/commits/7b28dae97f65bd76d98ff174ecf8c70252d99ec3>.

**Review by karnikaavelumani — APPROVED** [review]
LGTM :godmode:

**Pull request merged**